README.md

```diff
@@ -1,4 +1,261 @@
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To use this project, you will need a few tools;
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you will need a code editor, browser, git, NPM.
+
+<!--
+Example command:
+
+```code editor
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:<br>
+cd my-folder <br>
+git clone git@github.com:H-Ramalan/Hello-Microverse.git<br>
+cd my-project
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with: You can install the Microverse linters by following <a href="https://github.com/microverseinc/linters-config">these</a> steps.
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+live server
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/H-Ramalan)
+- Twitter: [@twitterhandle](https://twitter.com/hamza_ramalan)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+>
+
+If you like this project please you can support me by starring it on github.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+>
+
+I would like to thank my coding partner, my morning session team and also my stand up team for being very supportive during this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+>
+
+- **[What is the project about?]**
+
+  - [It is my first project at microverse]
+
+- **[will there be additional features?]**
+
+  - [Unfortunately, it is a simple project with very little chance of having any addedfeatures in the future.]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly consider adding a license to your repository. Public repositories on GitHub are often used to share open-source software. For your repository to be open source, you'll need to license it so that others are free to use, change, and distribute the software.

we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._